docs/manage-search-index.md

```diff
@@ -0,0 +1,50 @@
+# Manage Search Index
+
+Each document is stored in the index, and is the type and ID. The document is a JSON object that contains zero or more fields or key-value pairs. The original JSON document indexing will be stored in the field_source that returns a default receipt or document search.
```

Comment:
"and is"
has
"The original JSON document indexing will be stored in the field_source that returns a default receipt or document search." do not fully understand this phrase

Reply:
Neither I do understand it to be honest. It's exactly what I was talking about- technical stuff that was simply copy pasted from the sources I got .